programming/sr/servos/index.md

```diff
@@ -10,69 +10,72 @@ Servos
 This documentation refers to a feature which is only available on the physical robot kits.
 </div>
 
-The `servos` object is used to control a collection of Servo Boards.
-Similar to `motors` and `ruggeduinos`, `servos` can be used like a list.
-To do something with the **first Servo Board**, you would use:
+The `servo_board` object is used to control a collection of Servo Boards.
+
+When a single Servo Board is connected to your robot, you can control it
+using the `servo_board` object.
 
 ~~~~~ python
-R.servos[0].something...
+R.servo_board.something...
 ~~~~~
 
-...because indexes are 0-based (counting starts from 0, not 1).
-
-When you have more than one Servo Board connected to your kit
-they will be ordered based upon their serial number.
-
-The SR Part Code of each detected motor board is also printed to the log when your robot starts.
+The serial number of each detected Servo Board is printed to the log when your robot starts.
```

Comment:
Hrm, should we canonicalise around one of "part code" or "serial number"? I had previously assumed we were consistent, though it appears not. (The motor board docs mention "part code" I think).

Reply:
Discussed on the call, we concluded to canconicalise on serial number